tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "target": "es5",
+        "module": "commonjs",
+        "removeComments": true,
+        "experimentalDecorators": true,
+        "sourceMap": true
+    },
+    "files": [
+      "demo/node_modules/tns-core-modules/tns-core-modules.d.ts",
```

Comment:
We may remove this. It depends. It helps remove all warnings when building.
However, for first time setup (when a developer has not `npm install` inside the demo yet), it would not be available yet. Let me know what you guys think. I could go either way with this.